python/interpret_community/mimic/mimic_explainer.py

```diff
@@ -130,15 +130,15 @@ class MimicExplainer(BlackBoxExplainer):
     :type model_task: str
     :param reset_index: Uses the pandas DataFrame index column as part of the features when training
         the surrogate model.
-    :type reset_index: bool
+    :type reset_index: str
     """
 
     @init_tabular_decorator
     def __init__(self, model, initialization_examples, explainable_model, explainable_model_args=None,
                  is_function=False, augment_data=True, max_num_of_augmentations=10, explain_subset=None,
                  features=None, classes=None, transformations=None, allow_all_transformations=False,
                  shap_values_output=ShapValuesOutput.DEFAULT, categorical_features=None,
-                 model_task=ModelTask.Unknown, reset_index=False, **kwargs):
+                 model_task=ModelTask.Unknown, reset_index=ResetIndex.Ignore, **kwargs):
```

Comment:
Should this be used for other explainers as well? It might be nice to claim time-series support across the package.

Reply:
Eventually probably yes.  The other explainers also don't have datetime featurization yet.  Not sure I want to add more to other explainers at this point though as we are in the midst of a lot of discussion about where that code should live.